uint32 map_id
uint32 start_lane_id
---
static_centerline_optimizer/PointsWithLaneId[] points_with_lane_ids
string message
